bin/sl-svc-install.js

```diff
@@ -11,30 +13,19 @@ if (!opts.command && opts._.length > 0) {
 delete opts._;
 
 if (opts.help || opts.h) {
-  return usage(process.argv[1], console.log.bind(console));
+  return g.t('sl-service-install.txt',
+    process.argv[1], console.log.bind(console));
```

Comment:
I don't see this call signature for `t()` documented. Why is a function being passed in? Won't this cause the function source to be appended to the output?